src/ol_infrastructure/applications/airbyte/__main__.py

```diff
@@ -530,157 +558,564 @@
+)
+iam.RolePolicyAttachment(
+    f"airbyte-service-account-data-lake-access-policy-{env_name}",
+    policy_arn=data_lake_policy.arn,
+    role=airbyte_trust_role.role.name,
+)
+iam.RolePolicyAttachment(
+    f"airbyte-service-account-s3-source-access-policy-{env_name}",
+    policy_arn=s3_source_policy.arn,
+    role=airbyte_trust_role.role.name,
+)
+iam.RolePolicyAttachment(
+    "airbyte-service-account-policy-attachement",
+    policy_arn=airbyte_app_policy.arn,
+    role=airbyte_trust_role.role.name,
 )
 
-consul_datacenter = consul_stack.require_output("datacenter")
-grafana_credentials = read_yaml_secrets(
-    Path(f"vector/grafana.{stack_info.env_suffix}.yaml")
+airbyte_service_account_name = "airbyte-admin"
+vault_k8s_resources_config = OLVaultK8SResourcesConfig(
+    application_name="airbyte",
+    namespace=airbyte_namespace,
+    labels=k8s_global_labels,
+    vault_address=vault_config.require("address"),
+    vault_auth_endpoint=cluster_stack.require_output("vault_auth_endpoint"),
+    vault_auth_role_name=airbyte_vault_k8s_auth_backend_role.role_name,
+)
+vault_k8s_resources = OLVaultK8SResources(
+    resource_config=vault_k8s_resources_config,
+    opts=ResourceOptions(
+        provider=k8s_provider,
+        parent=k8s_provider,
+        delete_before_replace=True,
+        depends_on=[airbyte_vault_k8s_auth_backend_role],
+    ),
 )
 
-block_device_mappings = [BlockDeviceMapping(volume_size=500)]
-tag_specs = [
-    TagSpecification(
-        resource_type="instance",
-        tags=aws_config.merged_tags({"Name": airbyte_server_tag}),
+app_db_creds_secret_name = "app-db-creds"  # noqa: S105  # pragma: allowlist secret
+app_db_creds_dynamic_secret_config = OLVaultK8SDynamicSecretConfig(
+    name="airbyte-app-db-creds",
+    dest_secret_labels=k8s_global_labels,
+    dest_secret_name=app_db_creds_secret_name,
+    exclude_raw=True,
+    labels=k8s_global_labels,
+    mount=airbyte_db_vault_backend_config.mount_point,
+    namespace=airbyte_namespace,
+    path="creds/app",
+    templates={
+        "DATABASE_USER": '{{ get .Secrets "username" }}',
+        "DATABASE_PASSWORD": '{{  get .Secrets "password" }}',
+    },
+    vaultauth=vault_k8s_resources.auth_name,
+)
+app_db_creds_dynamic_secret = OLVaultK8SSecret(
+    "airbyte-app-db-creds-vaultdynamicsecret",
+    resource_config=app_db_creds_dynamic_secret_config,
+    opts=ResourceOptions(
+        provider=k8s_provider,
+        parent=k8s_provider,
     ),
-    TagSpecification(
-        resource_type="volume",
-        tags=aws_config.merged_tags({"Name": airbyte_server_tag}),
+)
+
+default_resources_definition = {
+    "limits": {
+        "cpu": "1000m",
+        "memory": "1Gi",
+    },
+    "requests": {
+        "cpu": "100m",
+        "memory": "1Gi",
+    },
+}
+
+airbyte_helm_release = kubernetes.helm.v3.Release(
+    "airbyte-helm-release",
+    kubernetes.helm.v3.ReleaseArgs(
+        name="airbyte",
+        chart="airbyte",
+        version="1.1.0",
```

Comment:
We should be pulling this from an environment variable and passing that in the pipeline, falling back to what is set in the version constants lib module.